packages/opc-header/src/opc-header.scss

```diff
@@ -0,0 +1,36 @@
+//@import "../../opc-styles/opc-styles.scss";
+
+$LOCAL: opc-header;
+
+$variables: ( 
+  //Add your variables here
+);
+
+:host {
+  --opc-header--width: 100%;
+}
+
+.app-header{
+  &__top-row{
+    &--header-name{
+      font-size: 1.5rem;
```

Comment:
@anjneeksharma wherever u've given the hardcoded values should be updated by scss variables from opc-styles package